dlc-wallet/src/main/scala/org/bitcoins/dlc/wallet/models/DLCCETSignatureDAO.scala

```diff
@@ -64,17 +66,16 @@ case class DLCCETSignatureDAO()(implicit
 
     def paramHash: Rep[Sha256DigestBE] = column("param_hash")
 
-    def outcomeHash: Rep[Sha256Digest] = column("outcome_hash")
+    def outcome: Rep[String] = column("outcome")
```

Comment:
this probably requires a migraiton

Reply:
It won't but old DLCs will no longer be valid